From bce81efdeac1cbefd0196b478a32aa2586bd595a Mon Sep 17 00:00:00 2001
From: Douwe Maan <hi@douwe.me>
Date: Fri, 18 Apr 2025 08:51:21 -0600
Subject: [PATCH] Do not provide `field_name` in validator core schemas
 (#11761)

Instead, the `field_name` is provided during validation by `pydantic-core` (hence the version bump).
---
 docs/concepts/types.md                 |   2 +-
 pydantic/_internal/_generate_schema.py |  45 +++---
 pydantic/functional_validators.py      |   5 +-
 pydantic/version.py                    |   2 +-
 pyproject.toml                         |   2 +-
 tests/test_validators.py               |  27 +++-
 uv.lock                                | 204 ++++++++++++-------------
 7 files changed, 148 insertions(+), 139 deletions(-)

diff --git a/docs/concepts/types.md b/docs/concepts/types.md
index d7cc290b81c..4b6ccfbaffd 100644
--- a/docs/concepts/types.md
+++ b/docs/concepts/types.md
@@ -979,7 +979,7 @@ class CustomType:
         cls, source_type: Any, handler: GetCoreSchemaHandler
     ) -> core_schema.CoreSchema:
         return core_schema.with_info_after_validator_function(
-            cls.validate, handler(int), field_name=handler.field_name
+            cls.validate, handler(int)
         )
 
 
diff --git a/pydantic/_internal/_generate_schema.py b/pydantic/_internal/_generate_schema.py
index 92069440ffa..afd623cd5c6 100644
--- a/pydantic/_internal/_generate_schema.py
+++ b/pydantic/_internal/_generate_schema.py
@@ -218,7 +218,6 @@ def filter_field_decorator_info_by_field(
 def apply_each_item_validators(
     schema: core_schema.CoreSchema,
     each_item_validators: list[Decorator[ValidatorDecoratorInfo]],
-    field_name: str | None,
 ) -> core_schema.CoreSchema:
     # This V1 compatibility shim should eventually be removed
 
@@ -230,21 +229,20 @@ def apply_each_item_validators(
     # note that this won't work for any Annotated types that get wrapped by a function validator
     # but that's okay because that didn't exist in V1
     if schema['type'] == 'nullable':
-        schema['schema'] = apply_each_item_validators(schema['schema'], each_item_validators, field_name)
+        schema['schema'] = apply_each_item_validators(schema['schema'], each_item_validators)
         return schema
     elif schema['type'] == 'tuple':
         if (variadic_item_index := schema.get('variadic_item_index')) is not None:
             schema['items_schema'][variadic_item_index] = apply_validators(
                 schema['items_schema'][variadic_item_index],
                 each_item_validators,
-                field_name,
             )
     elif is_list_like_schema_with_items_schema(schema):
         inner_schema = schema.get('items_schema', core_schema.any_schema())
-        schema['items_schema'] = apply_validators(inner_schema, each_item_validators, field_name)
+        schema['items_schema'] = apply_validators(inner_schema, each_item_validators)
     elif schema['type'] == 'dict':
         inner_schema = schema.get('values_schema', core_schema.any_schema())
-        schema['values_schema'] = apply_validators(inner_schema, each_item_validators, field_name)
+        schema['values_schema'] = apply_validators(inner_schema, each_item_validators)
     else:
         raise TypeError(
             f'`@validator(..., each_item=True)` cannot be applied to fields with a schema of {schema["type"]}'
@@ -823,7 +821,7 @@ def _model_schema(self, cls: type[BaseModel]) -> core_schema.CoreSchema:
                         extras_keys_schema=extras_keys_schema,
                         model_name=cls.__name__,
                     )
-                    inner_schema = apply_validators(fields_schema, decorators.root_validators.values(), None)
+                    inner_schema = apply_validators(fields_schema, decorators.root_validators.values())
                     inner_schema = apply_model_validators(inner_schema, model_validators, 'inner')
 
                     model_schema = core_schema.model_schema(
@@ -1363,9 +1361,9 @@ def set_discriminator(schema: CoreSchema) -> CoreSchema:
             field_info.validate_default = True
         each_item_validators = [v for v in this_field_validators if v.info.each_item is True]
         this_field_validators = [v for v in this_field_validators if v not in each_item_validators]
-        schema = apply_each_item_validators(schema, each_item_validators, name)
+        schema = apply_each_item_validators(schema, each_item_validators)
 
-        schema = apply_validators(schema, this_field_validators, name)
+        schema = apply_validators(schema, this_field_validators)
 
         # the default validator needs to go outside of any other validators
         # so that it is the topmost validator for the field validator
@@ -1942,7 +1940,7 @@ def _dataclass_schema(
                     collect_init_only=has_post_init,
                 )
 
-                inner_schema = apply_validators(args_schema, decorators.root_validators.values(), None)
+                inner_schema = apply_validators(args_schema, decorators.root_validators.values())
 
                 model_validators = decorators.model_validators.values()
                 inner_schema = apply_model_validators(inner_schema, model_validators, 'inner')
@@ -2454,24 +2452,16 @@ def _apply_model_serializers(
 
 _VALIDATOR_F_MATCH: Mapping[
     tuple[FieldValidatorModes, Literal['no-info', 'with-info']],
-    Callable[[Callable[..., Any], core_schema.CoreSchema, str | None], core_schema.CoreSchema],
+    Callable[[Callable[..., Any], core_schema.CoreSchema], core_schema.CoreSchema],
 ] = {
-    ('before', 'no-info'): lambda f, schema, _: core_schema.no_info_before_validator_function(f, schema),
-    ('after', 'no-info'): lambda f, schema, _: core_schema.no_info_after_validator_function(f, schema),
-    ('plain', 'no-info'): lambda f, _1, _2: core_schema.no_info_plain_validator_function(f),
-    ('wrap', 'no-info'): lambda f, schema, _: core_schema.no_info_wrap_validator_function(f, schema),
-    ('before', 'with-info'): lambda f, schema, field_name: core_schema.with_info_before_validator_function(
-        f, schema, field_name=field_name
-    ),
-    ('after', 'with-info'): lambda f, schema, field_name: core_schema.with_info_after_validator_function(
-        f, schema, field_name=field_name
-    ),
-    ('plain', 'with-info'): lambda f, _, field_name: core_schema.with_info_plain_validator_function(
-        f, field_name=field_name
-    ),
-    ('wrap', 'with-info'): lambda f, schema, field_name: core_schema.with_info_wrap_validator_function(
-        f, schema, field_name=field_name
-    ),
+    ('before', 'no-info'): lambda f, schema: core_schema.no_info_before_validator_function(f, schema),
+    ('after', 'no-info'): lambda f, schema: core_schema.no_info_after_validator_function(f, schema),
+    ('plain', 'no-info'): lambda f, _: core_schema.no_info_plain_validator_function(f),
+    ('wrap', 'no-info'): lambda f, schema: core_schema.no_info_wrap_validator_function(f, schema),
+    ('before', 'with-info'): lambda f, schema: core_schema.with_info_before_validator_function(f, schema),
+    ('after', 'with-info'): lambda f, schema: core_schema.with_info_after_validator_function(f, schema),
+    ('plain', 'with-info'): lambda f, _: core_schema.with_info_plain_validator_function(f),
+    ('wrap', 'with-info'): lambda f, schema: core_schema.with_info_wrap_validator_function(f, schema),
 }
 
 
@@ -2482,7 +2472,6 @@ def apply_validators(
     validators: Iterable[Decorator[RootValidatorDecoratorInfo]]
     | Iterable[Decorator[ValidatorDecoratorInfo]]
     | Iterable[Decorator[FieldValidatorDecoratorInfo]],
-    field_name: str | None,
 ) -> core_schema.CoreSchema:
     """Apply validators to a schema.
 
@@ -2498,7 +2487,7 @@ def apply_validators(
         info_arg = inspect_validator(validator.func, validator.info.mode)
         val_type = 'with-info' if info_arg else 'no-info'
 
-        schema = _VALIDATOR_F_MATCH[(validator.info.mode, val_type)](validator.func, schema, field_name)
+        schema = _VALIDATOR_F_MATCH[(validator.info.mode, val_type)](validator.func, schema)
     return schema
 
 
diff --git a/pydantic/functional_validators.py b/pydantic/functional_validators.py
index 2eed4ef8fc5..8b0ea318287 100644
--- a/pydantic/functional_validators.py
+++ b/pydantic/functional_validators.py
@@ -75,7 +75,7 @@ def __get_pydantic_core_schema__(self, source_type: Any, handler: GetCoreSchemaH
         info_arg = _inspect_validator(self.func, 'after')
         if info_arg:
             func = cast(core_schema.WithInfoValidatorFunction, self.func)
-            return core_schema.with_info_after_validator_function(func, schema=schema, field_name=handler.field_name)
+            return core_schema.with_info_after_validator_function(func, schema=schema)
         else:
             func = cast(core_schema.NoInfoValidatorFunction, self.func)
             return core_schema.no_info_after_validator_function(func, schema=schema)
@@ -136,7 +136,6 @@ def __get_pydantic_core_schema__(self, source_type: Any, handler: GetCoreSchemaH
             return core_schema.with_info_before_validator_function(
                 func,
                 schema=schema,
-                field_name=handler.field_name,
                 json_schema_input_schema=input_schema,
             )
         else:
@@ -230,7 +229,6 @@ def __get_pydantic_core_schema__(self, source_type: Any, handler: GetCoreSchemaH
             func = cast(core_schema.WithInfoValidatorFunction, self.func)
             return core_schema.with_info_plain_validator_function(
                 func,
-                field_name=handler.field_name,
                 serialization=serialization,  # pyright: ignore[reportArgumentType]
                 json_schema_input_schema=input_schema,
             )
@@ -307,7 +305,6 @@ def __get_pydantic_core_schema__(self, source_type: Any, handler: GetCoreSchemaH
             return core_schema.with_info_wrap_validator_function(
                 func,
                 schema=schema,
-                field_name=handler.field_name,
                 json_schema_input_schema=input_schema,
             )
         else:
diff --git a/pydantic/version.py b/pydantic/version.py
index 554a7a00c09..c50e241b76d 100644
--- a/pydantic/version.py
+++ b/pydantic/version.py
@@ -66,7 +66,7 @@ def version_info() -> str:
 def check_pydantic_core_version() -> bool:
     """Check that the installed `pydantic-core` dependency is compatible."""
     # Keep this in sync with the version constraint in the `pyproject.toml` dependencies:
-    return __pydantic_core_version__ == '2.33.1'
+    return __pydantic_core_version__ == '2.34.1'
 
 
 def parse_mypy_version(version: str) -> tuple[int, int, int]:
diff --git a/pyproject.toml b/pyproject.toml
index 8458badaaa8..d4cce4aef85 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -46,7 +46,7 @@ dependencies = [
     'typing-extensions>=4.12.2',
     'annotated-types>=0.6.0',
     # Keep this in sync with the version in the `check_pydantic_core_version()` function:
-    'pydantic-core==2.33.1',
+    'pydantic-core==2.34.1',
     'typing-inspection>=0.4.0',
 ]
 dynamic = ['version', 'readme']
diff --git a/tests/test_validators.py b/tests/test_validators.py
index fe7b6c45c9f..3b16a68d8d5 100644
--- a/tests/test_validators.py
+++ b/tests/test_validators.py
@@ -21,7 +21,7 @@
 import pytest
 from dirty_equals import HasRepr, IsInstance
 from pydantic_core import core_schema
-from typing_extensions import TypedDict
+from typing_extensions import TypeAliasType, TypedDict
 
 from pydantic import (
     BaseModel,
@@ -2684,7 +2684,7 @@ def foobar_validate(value: Any, info: core_schema.ValidationInfo):
 class Foobar:
     @classmethod
     def __get_pydantic_core_schema__(cls, source_type: Any, handler: GetCoreSchemaHandler) -> core_schema.CoreSchema:
-        return core_schema.with_info_plain_validator_function(foobar_validate, field_name=handler.field_name)
+        return core_schema.with_info_plain_validator_function(foobar_validate)
 
 
 def test_custom_type_field_name_model():
@@ -2779,6 +2779,29 @@ class MyModel(BaseModel):
     assert m.foobar == {'value': '1', 'field_name': 'foobar', 'data': {'x': 123}}
 
 
+def test_validator_field_name_with_reused_type_alias():
+    calls = []
+
+    def validate_my_field(value: str, info: ValidationInfo):
+        calls.append((info.field_name, value))
+        return value
+
+    MyField = TypeAliasType('MyField', Annotated[str, AfterValidator(validate_my_field)])
+
+    class MyModel(BaseModel):
+        field1: MyField
+        field2: MyField
+
+    MyModel.model_validate(
+        {
+            'field1': 'value1',
+            'field2': 'value2',
+        }
+    )
+
+    assert calls == [('field1', 'value1'), ('field2', 'value2')]
+
+
 def validate_wrap(value: Any, handler: core_schema.ValidatorFunctionWrapHandler, info: core_schema.ValidationInfo):
     data = info.data
     if isinstance(data, dict):
diff --git a/uv.lock b/uv.lock
index 2449038341f..59c572e4f9a 100644
--- a/uv.lock
+++ b/uv.lock
@@ -1784,7 +1784,7 @@ typechecking = [
 requires-dist = [
     { name = "annotated-types", specifier = ">=0.6.0" },
     { name = "email-validator", marker = "extra == 'email'", specifier = ">=2.0.0" },
-    { name = "pydantic-core", specifier = "==2.33.1" },
+    { name = "pydantic-core", specifier = "==2.34.1" },
     { name = "typing-extensions", specifier = ">=4.12.2" },
     { name = "typing-inspection", specifier = ">=0.4.0" },
     { name = "tzdata", marker = "python_full_version >= '3.9' and sys_platform == 'win32' and extra == 'timezone'" },
@@ -1881,111 +1881,111 @@ typechecking = [
 
 [[package]]
 name = "pydantic-core"
-version = "2.33.1"
+version = "2.34.1"
 source = { registry = "https://pypi.org/simple" }
 dependencies = [
     { name = "typing-extensions" },
 ]
-sdist = { url = "https://files.pythonhosted.org/packages/17/19/ed6a078a5287aea7922de6841ef4c06157931622c89c2a47940837b5eecd/pydantic_core-2.33.1.tar.gz", hash = "sha256:bcc9c6fdb0ced789245b02b7d6603e17d1563064ddcfc36f046b61c0c05dd9df", size = 434395 }
-wheels = [
-    { url = "https://files.pythonhosted.org/packages/38/ea/5f572806ab4d4223d11551af814d243b0e3e02cc6913def4d1fe4a5ca41c/pydantic_core-2.33.1-cp310-cp310-macosx_10_12_x86_64.whl", hash = "sha256:3077cfdb6125cc8dab61b155fdd714663e401f0e6883f9632118ec12cf42df26", size = 2044021 },
-    { url = "https://files.pythonhosted.org/packages/8c/d1/f86cc96d2aa80e3881140d16d12ef2b491223f90b28b9a911346c04ac359/pydantic_core-2.33.1-cp310-cp310-macosx_11_0_arm64.whl", hash = "sha256:8ffab8b2908d152e74862d276cf5017c81a2f3719f14e8e3e8d6b83fda863927", size = 1861742 },
-    { url = "https://files.pythonhosted.org/packages/37/08/fbd2cd1e9fc735a0df0142fac41c114ad9602d1c004aea340169ae90973b/pydantic_core-2.33.1-cp310-cp310-manylinux_2_17_aarch64.manylinux2014_aarch64.whl", hash = "sha256:5183e4f6a2d468787243ebcd70cf4098c247e60d73fb7d68d5bc1e1beaa0c4db", size = 1910414 },
-    { url = "https://files.pythonhosted.org/packages/7f/73/3ac217751decbf8d6cb9443cec9b9eb0130eeada6ae56403e11b486e277e/pydantic_core-2.33.1-cp310-cp310-manylinux_2_17_armv7l.manylinux2014_armv7l.whl", hash = "sha256:398a38d323f37714023be1e0285765f0a27243a8b1506b7b7de87b647b517e48", size = 1996848 },
-    { url = "https://files.pythonhosted.org/packages/9a/f5/5c26b265cdcff2661e2520d2d1e9db72d117ea00eb41e00a76efe68cb009/pydantic_core-2.33.1-cp310-cp310-manylinux_2_17_ppc64le.manylinux2014_ppc64le.whl", hash = "sha256:87d3776f0001b43acebfa86f8c64019c043b55cc5a6a2e313d728b5c95b46969", size = 2141055 },
-    { url = "https://files.pythonhosted.org/packages/5d/14/a9c3cee817ef2f8347c5ce0713e91867a0dceceefcb2973942855c917379/pydantic_core-2.33.1-cp310-cp310-manylinux_2_17_s390x.manylinux2014_s390x.whl", hash = "sha256:c566dd9c5f63d22226409553531f89de0cac55397f2ab8d97d6f06cfce6d947e", size = 2753806 },
-    { url = "https://files.pythonhosted.org/packages/f2/68/866ce83a51dd37e7c604ce0050ff6ad26de65a7799df89f4db87dd93d1d6/pydantic_core-2.33.1-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl", hash = "sha256:a0d5f3acc81452c56895e90643a625302bd6be351e7010664151cc55b7b97f89", size = 2007777 },
-    { url = "https://files.pythonhosted.org/packages/b6/a8/36771f4404bb3e49bd6d4344da4dede0bf89cc1e01f3b723c47248a3761c/pydantic_core-2.33.1-cp310-cp310-manylinux_2_5_i686.manylinux1_i686.whl", hash = "sha256:d3a07fadec2a13274a8d861d3d37c61e97a816beae717efccaa4b36dfcaadcde", size = 2122803 },
-    { url = "https://files.pythonhosted.org/packages/18/9c/730a09b2694aa89360d20756369822d98dc2f31b717c21df33b64ffd1f50/pydantic_core-2.33.1-cp310-cp310-musllinux_1_1_aarch64.whl", hash = "sha256:f99aeda58dce827f76963ee87a0ebe75e648c72ff9ba1174a253f6744f518f65", size = 2086755 },
-    { url = "https://files.pythonhosted.org/packages/54/8e/2dccd89602b5ec31d1c58138d02340ecb2ebb8c2cac3cc66b65ce3edb6ce/pydantic_core-2.33.1-cp310-cp310-musllinux_1_1_armv7l.whl", hash = "sha256:902dbc832141aa0ec374f4310f1e4e7febeebc3256f00dc359a9ac3f264a45dc", size = 2257358 },
-    { url = "https://files.pythonhosted.org/packages/d1/9c/126e4ac1bfad8a95a9837acdd0963695d69264179ba4ede8b8c40d741702/pydantic_core-2.33.1-cp310-cp310-musllinux_1_1_x86_64.whl", hash = "sha256:fe44d56aa0b00d66640aa84a3cbe80b7a3ccdc6f0b1ca71090696a6d4777c091", size = 2257916 },
-    { url = "https://files.pythonhosted.org/packages/7d/ba/91eea2047e681a6853c81c20aeca9dcdaa5402ccb7404a2097c2adf9d038/pydantic_core-2.33.1-cp310-cp310-win32.whl", hash = "sha256:ed3eb16d51257c763539bde21e011092f127a2202692afaeaccb50db55a31383", size = 1923823 },
-    { url = "https://files.pythonhosted.org/packages/94/c0/fcdf739bf60d836a38811476f6ecd50374880b01e3014318b6e809ddfd52/pydantic_core-2.33.1-cp310-cp310-win_amd64.whl", hash = "sha256:694ad99a7f6718c1a498dc170ca430687a39894a60327f548e02a9c7ee4b6504", size = 1952494 },
-    { url = "https://files.pythonhosted.org/packages/d6/7f/c6298830cb780c46b4f46bb24298d01019ffa4d21769f39b908cd14bbd50/pydantic_core-2.33.1-cp311-cp311-macosx_10_12_x86_64.whl", hash = "sha256:6e966fc3caaf9f1d96b349b0341c70c8d6573bf1bac7261f7b0ba88f96c56c24", size = 2044224 },
-    { url = "https://files.pythonhosted.org/packages/a8/65/6ab3a536776cad5343f625245bd38165d6663256ad43f3a200e5936afd6c/pydantic_core-2.33.1-cp311-cp311-macosx_11_0_arm64.whl", hash = "sha256:bfd0adeee563d59c598ceabddf2c92eec77abcb3f4a391b19aa7366170bd9e30", size = 1858845 },
-    { url = "https://files.pythonhosted.org/packages/e9/15/9a22fd26ba5ee8c669d4b8c9c244238e940cd5d818649603ca81d1c69861/pydantic_core-2.33.1-cp311-cp311-manylinux_2_17_aarch64.manylinux2014_aarch64.whl", hash = "sha256:91815221101ad3c6b507804178a7bb5cb7b2ead9ecd600041669c8d805ebd595", size = 1910029 },
-    { url = "https://files.pythonhosted.org/packages/d5/33/8cb1a62818974045086f55f604044bf35b9342900318f9a2a029a1bec460/pydantic_core-2.33.1-cp311-cp311-manylinux_2_17_armv7l.manylinux2014_armv7l.whl", hash = "sha256:9fea9c1869bb4742d174a57b4700c6dadea951df8b06de40c2fedb4f02931c2e", size = 1997784 },
-    { url = "https://files.pythonhosted.org/packages/c0/ca/49958e4df7715c71773e1ea5be1c74544923d10319173264e6db122543f9/pydantic_core-2.33.1-cp311-cp311-manylinux_2_17_ppc64le.manylinux2014_ppc64le.whl", hash = "sha256:1d20eb4861329bb2484c021b9d9a977566ab16d84000a57e28061151c62b349a", size = 2141075 },
-    { url = "https://files.pythonhosted.org/packages/7b/a6/0b3a167a9773c79ba834b959b4e18c3ae9216b8319bd8422792abc8a41b1/pydantic_core-2.33.1-cp311-cp311-manylinux_2_17_s390x.manylinux2014_s390x.whl", hash = "sha256:0fb935c5591573ae3201640579f30128ccc10739b45663f93c06796854405505", size = 2745849 },
-    { url = "https://files.pythonhosted.org/packages/0b/60/516484135173aa9e5861d7a0663dce82e4746d2e7f803627d8c25dfa5578/pydantic_core-2.33.1-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl", hash = "sha256:c964fd24e6166420d18fb53996d8c9fd6eac9bf5ae3ec3d03015be4414ce497f", size = 2005794 },
-    { url = "https://files.pythonhosted.org/packages/86/70/05b1eb77459ad47de00cf78ee003016da0cedf8b9170260488d7c21e9181/pydantic_core-2.33.1-cp311-cp311-manylinux_2_5_i686.manylinux1_i686.whl", hash = "sha256:681d65e9011f7392db5aa002b7423cc442d6a673c635668c227c6c8d0e5a4f77", size = 2123237 },
-    { url = "https://files.pythonhosted.org/packages/c7/57/12667a1409c04ae7dc95d3b43158948eb0368e9c790be8b095cb60611459/pydantic_core-2.33.1-cp311-cp311-musllinux_1_1_aarch64.whl", hash = "sha256:e100c52f7355a48413e2999bfb4e139d2977a904495441b374f3d4fb4a170961", size = 2086351 },
-    { url = "https://files.pythonhosted.org/packages/57/61/cc6d1d1c1664b58fdd6ecc64c84366c34ec9b606aeb66cafab6f4088974c/pydantic_core-2.33.1-cp311-cp311-musllinux_1_1_armv7l.whl", hash = "sha256:048831bd363490be79acdd3232f74a0e9951b11b2b4cc058aeb72b22fdc3abe1", size = 2258914 },
-    { url = "https://files.pythonhosted.org/packages/d1/0a/edb137176a1f5419b2ddee8bde6a0a548cfa3c74f657f63e56232df8de88/pydantic_core-2.33.1-cp311-cp311-musllinux_1_1_x86_64.whl", hash = "sha256:bdc84017d28459c00db6f918a7272a5190bec3090058334e43a76afb279eac7c", size = 2257385 },
-    { url = "https://files.pythonhosted.org/packages/26/3c/48ca982d50e4b0e1d9954919c887bdc1c2b462801bf408613ccc641b3daa/pydantic_core-2.33.1-cp311-cp311-win32.whl", hash = "sha256:32cd11c5914d1179df70406427097c7dcde19fddf1418c787540f4b730289896", size = 1923765 },
-    { url = "https://files.pythonhosted.org/packages/33/cd/7ab70b99e5e21559f5de38a0928ea84e6f23fdef2b0d16a6feaf942b003c/pydantic_core-2.33.1-cp311-cp311-win_amd64.whl", hash = "sha256:2ea62419ba8c397e7da28a9170a16219d310d2cf4970dbc65c32faf20d828c83", size = 1950688 },
-    { url = "https://files.pythonhosted.org/packages/4b/ae/db1fc237b82e2cacd379f63e3335748ab88b5adde98bf7544a1b1bd10a84/pydantic_core-2.33.1-cp311-cp311-win_arm64.whl", hash = "sha256:fc903512177361e868bc1f5b80ac8c8a6e05fcdd574a5fb5ffeac5a9982b9e89", size = 1908185 },
-    { url = "https://files.pythonhosted.org/packages/c8/ce/3cb22b07c29938f97ff5f5bb27521f95e2ebec399b882392deb68d6c440e/pydantic_core-2.33.1-cp312-cp312-macosx_10_12_x86_64.whl", hash = "sha256:1293d7febb995e9d3ec3ea09caf1a26214eec45b0f29f6074abb004723fc1de8", size = 2026640 },
-    { url = "https://files.pythonhosted.org/packages/19/78/f381d643b12378fee782a72126ec5d793081ef03791c28a0fd542a5bee64/pydantic_core-2.33.1-cp312-cp312-macosx_11_0_arm64.whl", hash = "sha256:99b56acd433386c8f20be5c4000786d1e7ca0523c8eefc995d14d79c7a081498", size = 1852649 },
-    { url = "https://files.pythonhosted.org/packages/9d/2b/98a37b80b15aac9eb2c6cfc6dbd35e5058a352891c5cce3a8472d77665a6/pydantic_core-2.33.1-cp312-cp312-manylinux_2_17_aarch64.manylinux2014_aarch64.whl", hash = "sha256:35a5ec3fa8c2fe6c53e1b2ccc2454398f95d5393ab398478f53e1afbbeb4d939", size = 1892472 },
-    { url = "https://files.pythonhosted.org/packages/4e/d4/3c59514e0f55a161004792b9ff3039da52448f43f5834f905abef9db6e4a/pydantic_core-2.33.1-cp312-cp312-manylinux_2_17_armv7l.manylinux2014_armv7l.whl", hash = "sha256:b172f7b9d2f3abc0efd12e3386f7e48b576ef309544ac3a63e5e9cdd2e24585d", size = 1977509 },
-    { url = "https://files.pythonhosted.org/packages/a9/b6/c2c7946ef70576f79a25db59a576bce088bdc5952d1b93c9789b091df716/pydantic_core-2.33.1-cp312-cp312-manylinux_2_17_ppc64le.manylinux2014_ppc64le.whl", hash = "sha256:9097b9f17f91eea659b9ec58148c0747ec354a42f7389b9d50701610d86f812e", size = 2128702 },
-    { url = "https://files.pythonhosted.org/packages/88/fe/65a880f81e3f2a974312b61f82a03d85528f89a010ce21ad92f109d94deb/pydantic_core-2.33.1-cp312-cp312-manylinux_2_17_s390x.manylinux2014_s390x.whl", hash = "sha256:cc77ec5b7e2118b152b0d886c7514a4653bcb58c6b1d760134a9fab915f777b3", size = 2679428 },
-    { url = "https://files.pythonhosted.org/packages/6f/ff/4459e4146afd0462fb483bb98aa2436d69c484737feaceba1341615fb0ac/pydantic_core-2.33.1-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl", hash = "sha256:d5e3d15245b08fa4a84cefc6c9222e6f37c98111c8679fbd94aa145f9a0ae23d", size = 2008753 },
-    { url = "https://files.pythonhosted.org/packages/7c/76/1c42e384e8d78452ededac8b583fe2550c84abfef83a0552e0e7478ccbc3/pydantic_core-2.33.1-cp312-cp312-manylinux_2_5_i686.manylinux1_i686.whl", hash = "sha256:ef99779001d7ac2e2461d8ab55d3373fe7315caefdbecd8ced75304ae5a6fc6b", size = 2114849 },
-    { url = "https://files.pythonhosted.org/packages/00/72/7d0cf05095c15f7ffe0eb78914b166d591c0eed72f294da68378da205101/pydantic_core-2.33.1-cp312-cp312-musllinux_1_1_aarch64.whl", hash = "sha256:fc6bf8869e193855e8d91d91f6bf59699a5cdfaa47a404e278e776dd7f168b39", size = 2069541 },
-    { url = "https://files.pythonhosted.org/packages/b3/69/94a514066bb7d8be499aa764926937409d2389c09be0b5107a970286ef81/pydantic_core-2.33.1-cp312-cp312-musllinux_1_1_armv7l.whl", hash = "sha256:b1caa0bc2741b043db7823843e1bde8aaa58a55a58fda06083b0569f8b45693a", size = 2239225 },
-    { url = "https://files.pythonhosted.org/packages/84/b0/e390071eadb44b41f4f54c3cef64d8bf5f9612c92686c9299eaa09e267e2/pydantic_core-2.33.1-cp312-cp312-musllinux_1_1_x86_64.whl", hash = "sha256:ec259f62538e8bf364903a7d0d0239447059f9434b284f5536e8402b7dd198db", size = 2248373 },
-    { url = "https://files.pythonhosted.org/packages/d6/b2/288b3579ffc07e92af66e2f1a11be3b056fe1214aab314748461f21a31c3/pydantic_core-2.33.1-cp312-cp312-win32.whl", hash = "sha256:e14f369c98a7c15772b9da98987f58e2b509a93235582838bd0d1d8c08b68fda", size = 1907034 },
-    { url = "https://files.pythonhosted.org/packages/02/28/58442ad1c22b5b6742b992ba9518420235adced665513868f99a1c2638a5/pydantic_core-2.33.1-cp312-cp312-win_amd64.whl", hash = "sha256:1c607801d85e2e123357b3893f82c97a42856192997b95b4d8325deb1cd0c5f4", size = 1956848 },
-    { url = "https://files.pythonhosted.org/packages/a1/eb/f54809b51c7e2a1d9f439f158b8dd94359321abcc98767e16fc48ae5a77e/pydantic_core-2.33.1-cp312-cp312-win_arm64.whl", hash = "sha256:8d13f0276806ee722e70a1c93da19748594f19ac4299c7e41237fc791d1861ea", size = 1903986 },
-    { url = "https://files.pythonhosted.org/packages/7a/24/eed3466a4308d79155f1cdd5c7432c80ddcc4530ba8623b79d5ced021641/pydantic_core-2.33.1-cp313-cp313-macosx_10_12_x86_64.whl", hash = "sha256:70af6a21237b53d1fe7b9325b20e65cbf2f0a848cf77bed492b029139701e66a", size = 2033551 },
-    { url = "https://files.pythonhosted.org/packages/ab/14/df54b1a0bc9b6ded9b758b73139d2c11b4e8eb43e8ab9c5847c0a2913ada/pydantic_core-2.33.1-cp313-cp313-macosx_11_0_arm64.whl", hash = "sha256:282b3fe1bbbe5ae35224a0dbd05aed9ccabccd241e8e6b60370484234b456266", size = 1852785 },
-    { url = "https://files.pythonhosted.org/packages/fa/96/e275f15ff3d34bb04b0125d9bc8848bf69f25d784d92a63676112451bfb9/pydantic_core-2.33.1-cp313-cp313-manylinux_2_17_aarch64.manylinux2014_aarch64.whl", hash = "sha256:4b315e596282bbb5822d0c7ee9d255595bd7506d1cb20c2911a4da0b970187d3", size = 1897758 },
-    { url = "https://files.pythonhosted.org/packages/b7/d8/96bc536e975b69e3a924b507d2a19aedbf50b24e08c80fb00e35f9baaed8/pydantic_core-2.33.1-cp313-cp313-manylinux_2_17_armv7l.manylinux2014_armv7l.whl", hash = "sha256:1dfae24cf9921875ca0ca6a8ecb4bb2f13c855794ed0d468d6abbec6e6dcd44a", size = 1986109 },
-    { url = "https://files.pythonhosted.org/packages/90/72/ab58e43ce7e900b88cb571ed057b2fcd0e95b708a2e0bed475b10130393e/pydantic_core-2.33.1-cp313-cp313-manylinux_2_17_ppc64le.manylinux2014_ppc64le.whl", hash = "sha256:6dd8ecfde08d8bfadaea669e83c63939af76f4cf5538a72597016edfa3fad516", size = 2129159 },
-    { url = "https://files.pythonhosted.org/packages/dc/3f/52d85781406886c6870ac995ec0ba7ccc028b530b0798c9080531b409fdb/pydantic_core-2.33.1-cp313-cp313-manylinux_2_17_s390x.manylinux2014_s390x.whl", hash = "sha256:2f593494876eae852dc98c43c6f260f45abdbfeec9e4324e31a481d948214764", size = 2680222 },
-    { url = "https://files.pythonhosted.org/packages/f4/56/6e2ef42f363a0eec0fd92f74a91e0ac48cd2e49b695aac1509ad81eee86a/pydantic_core-2.33.1-cp313-cp313-manylinux_2_17_x86_64.manylinux2014_x86_64.whl", hash = "sha256:948b73114f47fd7016088e5186d13faf5e1b2fe83f5e320e371f035557fd264d", size = 2006980 },
-    { url = "https://files.pythonhosted.org/packages/4c/c0/604536c4379cc78359f9ee0aa319f4aedf6b652ec2854953f5a14fc38c5a/pydantic_core-2.33.1-cp313-cp313-manylinux_2_5_i686.manylinux1_i686.whl", hash = "sha256:e11f3864eb516af21b01e25fac915a82e9ddad3bb0fb9e95a246067398b435a4", size = 2120840 },
-    { url = "https://files.pythonhosted.org/packages/1f/46/9eb764814f508f0edfb291a0f75d10854d78113fa13900ce13729aaec3ae/pydantic_core-2.33.1-cp313-cp313-musllinux_1_1_aarch64.whl", hash = "sha256:549150be302428b56fdad0c23c2741dcdb5572413776826c965619a25d9c6bde", size = 2072518 },
-    { url = "https://files.pythonhosted.org/packages/42/e3/fb6b2a732b82d1666fa6bf53e3627867ea3131c5f39f98ce92141e3e3dc1/pydantic_core-2.33.1-cp313-cp313-musllinux_1_1_armv7l.whl", hash = "sha256:495bc156026efafd9ef2d82372bd38afce78ddd82bf28ef5276c469e57c0c83e", size = 2248025 },
-    { url = "https://files.pythonhosted.org/packages/5c/9d/fbe8fe9d1aa4dac88723f10a921bc7418bd3378a567cb5e21193a3c48b43/pydantic_core-2.33.1-cp313-cp313-musllinux_1_1_x86_64.whl", hash = "sha256:ec79de2a8680b1a67a07490bddf9636d5c2fab609ba8c57597e855fa5fa4dacd", size = 2254991 },
-    { url = "https://files.pythonhosted.org/packages/aa/99/07e2237b8a66438d9b26482332cda99a9acccb58d284af7bc7c946a42fd3/pydantic_core-2.33.1-cp313-cp313-win32.whl", hash = "sha256:ee12a7be1742f81b8a65b36c6921022301d466b82d80315d215c4c691724986f", size = 1915262 },
-    { url = "https://files.pythonhosted.org/packages/8a/f4/e457a7849beeed1e5defbcf5051c6f7b3c91a0624dd31543a64fc9adcf52/pydantic_core-2.33.1-cp313-cp313-win_amd64.whl", hash = "sha256:ede9b407e39949d2afc46385ce6bd6e11588660c26f80576c11c958e6647bc40", size = 1956626 },
-    { url = "https://files.pythonhosted.org/packages/20/d0/e8d567a7cff7b04e017ae164d98011f1e1894269fe8e90ea187a3cbfb562/pydantic_core-2.33.1-cp313-cp313-win_arm64.whl", hash = "sha256:aa687a23d4b7871a00e03ca96a09cad0f28f443690d300500603bd0adba4b523", size = 1909590 },
-    { url = "https://files.pythonhosted.org/packages/ef/fd/24ea4302d7a527d672c5be06e17df16aabfb4e9fdc6e0b345c21580f3d2a/pydantic_core-2.33.1-cp313-cp313t-macosx_11_0_arm64.whl", hash = "sha256:401d7b76e1000d0dd5538e6381d28febdcacb097c8d340dde7d7fc6e13e9f95d", size = 1812963 },
-    { url = "https://files.pythonhosted.org/packages/5f/95/4fbc2ecdeb5c1c53f1175a32d870250194eb2fdf6291b795ab08c8646d5d/pydantic_core-2.33.1-cp313-cp313t-manylinux_2_17_x86_64.manylinux2014_x86_64.whl", hash = "sha256:7aeb055a42d734c0255c9e489ac67e75397d59c6fbe60d155851e9782f276a9c", size = 1986896 },
-    { url = "https://files.pythonhosted.org/packages/71/ae/fe31e7f4a62431222d8f65a3bd02e3fa7e6026d154a00818e6d30520ea77/pydantic_core-2.33.1-cp313-cp313t-win_amd64.whl", hash = "sha256:338ea9b73e6e109f15ab439e62cb3b78aa752c7fd9536794112e14bee02c8d18", size = 1931810 },
-    { url = "https://files.pythonhosted.org/packages/49/78/b86bad645cc3e8dfa6858c70ec38939bf350e54004837c48de09474b2b9e/pydantic_core-2.33.1-cp39-cp39-macosx_10_12_x86_64.whl", hash = "sha256:5ab77f45d33d264de66e1884fca158bc920cb5e27fd0764a72f72f5756ae8bdb", size = 2044282 },
-    { url = "https://files.pythonhosted.org/packages/3b/00/a02531331773b2bf08743d84c6b776bd6a449d23b3ae6b0e3229d568bac4/pydantic_core-2.33.1-cp39-cp39-macosx_11_0_arm64.whl", hash = "sha256:e7aaba1b4b03aaea7bb59e1b5856d734be011d3e6d98f5bcaa98cb30f375f2ad", size = 1877598 },
-    { url = "https://files.pythonhosted.org/packages/a1/fa/32cc152b84a1f420f8a7d80161373e8d87d4ffa077e67d6c8aab3ce1a6ab/pydantic_core-2.33.1-cp39-cp39-manylinux_2_17_aarch64.manylinux2014_aarch64.whl", hash = "sha256:7fb66263e9ba8fea2aa85e1e5578980d127fb37d7f2e292773e7bc3a38fb0c7b", size = 1911021 },
-    { url = "https://files.pythonhosted.org/packages/5e/87/ea553e0d98bce6c4876f8c50f65cb45597eff6e0aaa8b15813e9972bb19d/pydantic_core-2.33.1-cp39-cp39-manylinux_2_17_armv7l.manylinux2014_armv7l.whl", hash = "sha256:3f2648b9262607a7fb41d782cc263b48032ff7a03a835581abbf7a3bec62bcf5", size = 1997276 },
-    { url = "https://files.pythonhosted.org/packages/f7/9b/60cb9f4b52158b3adac0066492bbadd0b8473f4f8da5bcc73972655b76ef/pydantic_core-2.33.1-cp39-cp39-manylinux_2_17_ppc64le.manylinux2014_ppc64le.whl", hash = "sha256:723c5630c4259400818b4ad096735a829074601805d07f8cafc366d95786d331", size = 2141348 },
-    { url = "https://files.pythonhosted.org/packages/9b/38/374d254e270d4de0add68a8239f4ed0f444fdd7b766ea69244fb9491dccb/pydantic_core-2.33.1-cp39-cp39-manylinux_2_17_s390x.manylinux2014_s390x.whl", hash = "sha256:d100e3ae783d2167782391e0c1c7a20a31f55f8015f3293647544df3f9c67824", size = 2753708 },
-    { url = "https://files.pythonhosted.org/packages/05/a8/fd79111eb5ab9bc4ef98d8fb0b3a2ffdc80107b2c59859a741ab379c96f8/pydantic_core-2.33.1-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl", hash = "sha256:177d50460bc976a0369920b6c744d927b0ecb8606fb56858ff542560251b19e5", size = 2008699 },
-    { url = "https://files.pythonhosted.org/packages/35/31/2e06619868eb4c18642c5601db420599c1cf9cf50fe868c9ac09cd298e24/pydantic_core-2.33.1-cp39-cp39-manylinux_2_5_i686.manylinux1_i686.whl", hash = "sha256:a3edde68d1a1f9af1273b2fe798997b33f90308fb6d44d8550c89fc6a3647cf6", size = 2123426 },
-    { url = "https://files.pythonhosted.org/packages/4a/d0/3531e8783a311802e3db7ee5a1a5ed79e5706e930b1b4e3109ce15eeb681/pydantic_core-2.33.1-cp39-cp39-musllinux_1_1_aarch64.whl", hash = "sha256:a62c3c3ef6a7e2c45f7853b10b5bc4ddefd6ee3cd31024754a1a5842da7d598d", size = 2087330 },
-    { url = "https://files.pythonhosted.org/packages/ac/32/5ff252ed73bacd7677a706ab17723e261a76793f98b305aa20cfc10bbd56/pydantic_core-2.33.1-cp39-cp39-musllinux_1_1_armv7l.whl", hash = "sha256:c91dbb0ab683fa0cd64a6e81907c8ff41d6497c346890e26b23de7ee55353f96", size = 2258171 },
-    { url = "https://files.pythonhosted.org/packages/c9/f9/e96e00f92b8f5b3e2cddc80c5ee6cf038f8a0f238c44b67b01759943a7b4/pydantic_core-2.33.1-cp39-cp39-musllinux_1_1_x86_64.whl", hash = "sha256:9f466e8bf0a62dc43e068c12166281c2eca72121dd2adc1040f3aa1e21ef8599", size = 2258745 },
-    { url = "https://files.pythonhosted.org/packages/54/1e/51c86688e809d94797fdf0efc41514f001caec982a05f62d90c180a9639d/pydantic_core-2.33.1-cp39-cp39-win32.whl", hash = "sha256:ab0277cedb698749caada82e5d099dc9fed3f906a30d4c382d1a21725777a1e5", size = 1923626 },
-    { url = "https://files.pythonhosted.org/packages/57/18/c2da959fd8d019b70cadafdda2bf845378ada47973e0bad6cc84f56dbe6e/pydantic_core-2.33.1-cp39-cp39-win_amd64.whl", hash = "sha256:5773da0ee2d17136b1f1c6fbde543398d452a6ad2a7b54ea1033e2daa739b8d2", size = 1953703 },
-    { url = "https://files.pythonhosted.org/packages/9c/c7/8b311d5adb0fe00a93ee9b4e92a02b0ec08510e9838885ef781ccbb20604/pydantic_core-2.33.1-pp310-pypy310_pp73-macosx_10_12_x86_64.whl", hash = "sha256:5c834f54f8f4640fd7e4b193f80eb25a0602bba9e19b3cd2fc7ffe8199f5ae02", size = 2041659 },
-    { url = "https://files.pythonhosted.org/packages/8a/d6/4f58d32066a9e26530daaf9adc6664b01875ae0691570094968aaa7b8fcc/pydantic_core-2.33.1-pp310-pypy310_pp73-macosx_11_0_arm64.whl", hash = "sha256:049e0de24cf23766f12cc5cc71d8abc07d4a9deb9061b334b62093dedc7cb068", size = 1873294 },
-    { url = "https://files.pythonhosted.org/packages/f7/3f/53cc9c45d9229da427909c751f8ed2bf422414f7664ea4dde2d004f596ba/pydantic_core-2.33.1-pp310-pypy310_pp73-manylinux_2_17_aarch64.manylinux2014_aarch64.whl", hash = "sha256:1a28239037b3d6f16916a4c831a5a0eadf856bdd6d2e92c10a0da3a59eadcf3e", size = 1903771 },
-    { url = "https://files.pythonhosted.org/packages/f0/49/bf0783279ce674eb9903fb9ae43f6c614cb2f1c4951370258823f795368b/pydantic_core-2.33.1-pp310-pypy310_pp73-manylinux_2_17_x86_64.manylinux2014_x86_64.whl", hash = "sha256:9d3da303ab5f378a268fa7d45f37d7d85c3ec19769f28d2cc0c61826a8de21fe", size = 2083558 },
-    { url = "https://files.pythonhosted.org/packages/9c/5b/0d998367687f986c7d8484a2c476d30f07bf5b8b1477649a6092bd4c540e/pydantic_core-2.33.1-pp310-pypy310_pp73-manylinux_2_5_i686.manylinux1_i686.whl", hash = "sha256:25626fb37b3c543818c14821afe0fd3830bc327a43953bc88db924b68c5723f1", size = 2118038 },
-    { url = "https://files.pythonhosted.org/packages/b3/33/039287d410230ee125daee57373ac01940d3030d18dba1c29cd3089dc3ca/pydantic_core-2.33.1-pp310-pypy310_pp73-musllinux_1_1_aarch64.whl", hash = "sha256:3ab2d36e20fbfcce8f02d73c33a8a7362980cff717926bbae030b93ae46b56c7", size = 2079315 },
-    { url = "https://files.pythonhosted.org/packages/1f/85/6d8b2646d99c062d7da2d0ab2faeb0d6ca9cca4c02da6076376042a20da3/pydantic_core-2.33.1-pp310-pypy310_pp73-musllinux_1_1_armv7l.whl", hash = "sha256:2f9284e11c751b003fd4215ad92d325d92c9cb19ee6729ebd87e3250072cdcde", size = 2249063 },
-    { url = "https://files.pythonhosted.org/packages/17/d7/c37d208d5738f7b9ad8f22ae8a727d88ebf9c16c04ed2475122cc3f7224a/pydantic_core-2.33.1-pp310-pypy310_pp73-musllinux_1_1_x86_64.whl", hash = "sha256:048c01eee07d37cbd066fc512b9d8b5ea88ceeb4e629ab94b3e56965ad655add", size = 2254631 },
-    { url = "https://files.pythonhosted.org/packages/13/e0/bafa46476d328e4553b85ab9b2f7409e7aaef0ce4c937c894821c542d347/pydantic_core-2.33.1-pp310-pypy310_pp73-win_amd64.whl", hash = "sha256:5ccd429694cf26af7997595d627dd2637e7932214486f55b8a357edaac9dae8c", size = 2080877 },
-    { url = "https://files.pythonhosted.org/packages/0b/76/1794e440c1801ed35415238d2c728f26cd12695df9057154ad768b7b991c/pydantic_core-2.33.1-pp311-pypy311_pp73-macosx_10_12_x86_64.whl", hash = "sha256:3a371dc00282c4b84246509a5ddc808e61b9864aa1eae9ecc92bb1268b82db4a", size = 2042858 },
-    { url = "https://files.pythonhosted.org/packages/73/b4/9cd7b081fb0b1b4f8150507cd59d27b275c3e22ad60b35cb19ea0977d9b9/pydantic_core-2.33.1-pp311-pypy311_pp73-macosx_11_0_arm64.whl", hash = "sha256:f59295ecc75a1788af8ba92f2e8c6eeaa5a94c22fc4d151e8d9638814f85c8fc", size = 1873745 },
-    { url = "https://files.pythonhosted.org/packages/e1/d7/9ddb7575d4321e40d0363903c2576c8c0c3280ebea137777e5ab58d723e3/pydantic_core-2.33.1-pp311-pypy311_pp73-manylinux_2_17_aarch64.manylinux2014_aarch64.whl", hash = "sha256:08530b8ac922003033f399128505f513e30ca770527cc8bbacf75a84fcc2c74b", size = 1904188 },
-    { url = "https://files.pythonhosted.org/packages/d1/a8/3194ccfe461bb08da19377ebec8cb4f13c9bd82e13baebc53c5c7c39a029/pydantic_core-2.33.1-pp311-pypy311_pp73-manylinux_2_17_x86_64.manylinux2014_x86_64.whl", hash = "sha256:bae370459da6a5466978c0eacf90690cb57ec9d533f8e63e564ef3822bfa04fe", size = 2083479 },
-    { url = "https://files.pythonhosted.org/packages/42/c7/84cb569555d7179ca0b3f838cef08f66f7089b54432f5b8599aac6e9533e/pydantic_core-2.33.1-pp311-pypy311_pp73-manylinux_2_5_i686.manylinux1_i686.whl", hash = "sha256:e3de2777e3b9f4d603112f78006f4ae0acb936e95f06da6cb1a45fbad6bdb4b5", size = 2118415 },
-    { url = "https://files.pythonhosted.org/packages/3b/67/72abb8c73e0837716afbb58a59cc9e3ae43d1aa8677f3b4bc72c16142716/pydantic_core-2.33.1-pp311-pypy311_pp73-musllinux_1_1_aarch64.whl", hash = "sha256:3a64e81e8cba118e108d7126362ea30e021291b7805d47e4896e52c791be2761", size = 2079623 },
-    { url = "https://files.pythonhosted.org/packages/0b/cd/c59707e35a47ba4cbbf153c3f7c56420c58653b5801b055dc52cccc8e2dc/pydantic_core-2.33.1-pp311-pypy311_pp73-musllinux_1_1_armv7l.whl", hash = "sha256:52928d8c1b6bda03cc6d811e8923dffc87a2d3c8b3bfd2ce16471c7147a24850", size = 2250175 },
-    { url = "https://files.pythonhosted.org/packages/84/32/e4325a6676b0bed32d5b084566ec86ed7fd1e9bcbfc49c578b1755bde920/pydantic_core-2.33.1-pp311-pypy311_pp73-musllinux_1_1_x86_64.whl", hash = "sha256:1b30d92c9412beb5ac6b10a3eb7ef92ccb14e3f2a8d7732e2d739f58b3aa7544", size = 2254674 },
-    { url = "https://files.pythonhosted.org/packages/12/6f/5596dc418f2e292ffc661d21931ab34591952e2843e7168ea5a52591f6ff/pydantic_core-2.33.1-pp311-pypy311_pp73-win_amd64.whl", hash = "sha256:f995719707e0e29f0f41a8aa3bcea6e761a36c9136104d3189eafb83f5cec5e5", size = 2080951 },
-    { url = "https://files.pythonhosted.org/packages/2d/a8/c2c8f29bd18f7ef52de32a6deb9e3ee87ba18b7b2122636aa9f4438cf627/pydantic_core-2.33.1-pp39-pypy39_pp73-macosx_10_12_x86_64.whl", hash = "sha256:7edbc454a29fc6aeae1e1eecba4f07b63b8d76e76a748532233c4c167b4cb9ea", size = 2041791 },
-    { url = "https://files.pythonhosted.org/packages/08/ad/328081b1c82543ae49d0650048305058583c51f1a9a56a0d6e87bb3a2443/pydantic_core-2.33.1-pp39-pypy39_pp73-macosx_11_0_arm64.whl", hash = "sha256:ad05b683963f69a1d5d2c2bdab1274a31221ca737dbbceaa32bcb67359453cdd", size = 1873579 },
-    { url = "https://files.pythonhosted.org/packages/6e/8a/bc65dbf7e501e88367cdab06a2c1340457c785f0c72288cae737fd80c0fa/pydantic_core-2.33.1-pp39-pypy39_pp73-manylinux_2_17_aarch64.manylinux2014_aarch64.whl", hash = "sha256:df6a94bf9452c6da9b5d76ed229a5683d0306ccb91cca8e1eea883189780d568", size = 1904189 },
-    { url = "https://files.pythonhosted.org/packages/9a/db/30ca6aefda211fb01ef185ca73cb7a0c6e7fe952c524025c8782b5acd771/pydantic_core-2.33.1-pp39-pypy39_pp73-manylinux_2_17_x86_64.manylinux2014_x86_64.whl", hash = "sha256:7965c13b3967909a09ecc91f21d09cfc4576bf78140b988904e94f130f188396", size = 2084446 },
-    { url = "https://files.pythonhosted.org/packages/f2/89/a12b55286e30c9f476eab7c53c9249ec76faf70430596496ab0309f28629/pydantic_core-2.33.1-pp39-pypy39_pp73-manylinux_2_5_i686.manylinux1_i686.whl", hash = "sha256:3f1fdb790440a34f6ecf7679e1863b825cb5ffde858a9197f851168ed08371e5", size = 2118215 },
-    { url = "https://files.pythonhosted.org/packages/8e/55/12721c4a8d7951584ad3d9848b44442559cf1876e0bb424148d1060636b3/pydantic_core-2.33.1-pp39-pypy39_pp73-musllinux_1_1_aarch64.whl", hash = "sha256:5277aec8d879f8d05168fdd17ae811dd313b8ff894aeeaf7cd34ad28b4d77e33", size = 2079963 },
-    { url = "https://files.pythonhosted.org/packages/bd/0c/3391bd5d6ff62ea998db94732528d9bc32c560b0ed861c39119759461946/pydantic_core-2.33.1-pp39-pypy39_pp73-musllinux_1_1_armv7l.whl", hash = "sha256:8ab581d3530611897d863d1a649fb0644b860286b4718db919bfd51ece41f10b", size = 2249388 },
-    { url = "https://files.pythonhosted.org/packages/d3/5f/3e4feb042998d7886a9b523b372d83955cbc192a07013dcd24276db078ee/pydantic_core-2.33.1-pp39-pypy39_pp73-musllinux_1_1_x86_64.whl", hash = "sha256:0483847fa9ad5e3412265c1bd72aad35235512d9ce9d27d81a56d935ef489672", size = 2255226 },
-    { url = "https://files.pythonhosted.org/packages/25/f2/1647933efaaad61846109a27619f3704929e758a09e6431b8f932a053d40/pydantic_core-2.33.1-pp39-pypy39_pp73-win_amd64.whl", hash = "sha256:de9e06abe3cc5ec6a2d5f75bc99b0bdca4f5c719a5b34026f8c57efbdecd2ee3", size = 2081073 },
+sdist = { url = "https://files.pythonhosted.org/packages/42/fa/5f682f3db14e14f7b5b7f5008ac24594b93943d1da4facbacff5876624cf/pydantic_core-2.34.1.tar.gz", hash = "sha256:6bf31628ab6d0e7c7c0372419898c52ef0a447b33ab47c7f62053bd013cc5b09", size = 435281 }
+wheels = [
+    { url = "https://files.pythonhosted.org/packages/54/10/d950f249c9e50048890a515a4bad7e98c44ec4cde78ff4d6e37bfa022a19/pydantic_core-2.34.1-cp310-cp310-macosx_10_12_x86_64.whl", hash = "sha256:54f6bd146247c62b243fae34b535069614b255453fb1405c17b7d01d00976af7", size = 2038087 },
+    { url = "https://files.pythonhosted.org/packages/f5/6c/9177d1b5f9aac2775979b5d0a7f03589919dba05caf4d11d4ba9292ce105/pydantic_core-2.34.1-cp310-cp310-macosx_11_0_arm64.whl", hash = "sha256:9db4dab51a6515d7eb0763f200f4784075952b2a0afe84ce7048da3da70a51cb", size = 1864646 },
+    { url = "https://files.pythonhosted.org/packages/0a/36/8428d9848cfec707247d7d082deda2d20c8a6be8939c70adb228d9f9251d/pydantic_core-2.34.1-cp310-cp310-manylinux_2_17_aarch64.manylinux2014_aarch64.whl", hash = "sha256:c1444714e5c04dc20ad8246f8fd32b7ef4aef34f2fc7b349f71ae365ceb51e66", size = 1906297 },
+    { url = "https://files.pythonhosted.org/packages/74/53/d0c33441434a3c2ea79ad5b0c8063f5c55a1173ecf38a0a5632c18205df3/pydantic_core-2.34.1-cp310-cp310-manylinux_2_17_armv7l.manylinux2014_armv7l.whl", hash = "sha256:fffe3dd67708719a59e82ae36d6b431e2fb4e11b14f36f173c93e448ba963825", size = 1991861 },
+    { url = "https://files.pythonhosted.org/packages/35/9b/e19d326eeb1bf83f1f9d5f98a1e244da3441258824ae410b0dfb06c1ab5b/pydantic_core-2.34.1-cp310-cp310-manylinux_2_17_ppc64le.manylinux2014_ppc64le.whl", hash = "sha256:91f64f2f538d8da4a61d1d7dd515dfa535375bddfa1de0b543af650f2a9ae0a8", size = 2142095 },
+    { url = "https://files.pythonhosted.org/packages/de/25/b68f48ff4b3a7e3b3fb23cc1cd0dd12c024eabfa3641116dc984ff92222f/pydantic_core-2.34.1-cp310-cp310-manylinux_2_17_s390x.manylinux2014_s390x.whl", hash = "sha256:8d17f447fffbef877aab4bca91c65d0ccdb3ef776c3db24f88c499565568a489", size = 2745180 },
+    { url = "https://files.pythonhosted.org/packages/76/07/671c2c3876bd5f8a446d8264bc4832d23d2589786c6bd132b29181a1aa4a/pydantic_core-2.34.1-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl", hash = "sha256:8a44cd4d7651a586344cdd474e0740a8d2ee9dba9d1169f980b479b14a8b4891", size = 2015014 },
+    { url = "https://files.pythonhosted.org/packages/0b/33/47970d638c0db6b1087fc8ecaae0abc1cb67c7010e316560d1ef6ab4c664/pydantic_core-2.34.1-cp310-cp310-manylinux_2_5_i686.manylinux1_i686.whl", hash = "sha256:564b56ce0e64a48ec338b06205b8753cc648133ec3f9c9af9dcfd403fc7af040", size = 2120837 },
+    { url = "https://files.pythonhosted.org/packages/76/0b/7ce9448cfe087d5a4a084d067c317b92a631e10c6ed4a11687bb841f6296/pydantic_core-2.34.1-cp310-cp310-musllinux_1_1_aarch64.whl", hash = "sha256:def02d3a0296a171d6723f97cc135ae827bdaca24c9f0665d23ad6f8096334d1", size = 2083217 },
+    { url = "https://files.pythonhosted.org/packages/47/ca/13426f6593baff1dd544f6f884ce03f43133ef3787640e816fdc8ac5ec1f/pydantic_core-2.34.1-cp310-cp310-musllinux_1_1_armv7l.whl", hash = "sha256:b72c63ed0399fdb7f45b7985025aa851d6c964801b1ad7c58f3ce8f75bc4e070", size = 2254441 },
+    { url = "https://files.pythonhosted.org/packages/7c/7d/ecd2d9661000981f511ec996548024abdd3fcdd37a3582d4f30572f94651/pydantic_core-2.34.1-cp310-cp310-musllinux_1_1_x86_64.whl", hash = "sha256:eae6a05d4612d15ba5e22879b710d770d310dbdac6edc53126b86a3be4fe78b5", size = 2252748 },
+    { url = "https://files.pythonhosted.org/packages/17/2f/baeda4059c46361a917973fb938e6564c0b9970113dba4f126c2dcc32595/pydantic_core-2.34.1-cp310-cp310-win32.whl", hash = "sha256:51aae06a854be524f7a33fd2ccc54a37577a7b37c77813794483cbb00829d646", size = 1918465 },
+    { url = "https://files.pythonhosted.org/packages/e3/93/156001796df52ed6738fe1cd394a6741737260fec318ab4b97085edd9909/pydantic_core-2.34.1-cp310-cp310-win_amd64.whl", hash = "sha256:4f3044c1626b46d5c21e67d9fd16fd854e2847b23e7ebda75689733050959023", size = 1960435 },
+    { url = "https://files.pythonhosted.org/packages/c7/48/79d53d1822736d8304adb7bf12842c880a576fa53e670a4e0bcc9ebc7458/pydantic_core-2.34.1-cp311-cp311-macosx_10_12_x86_64.whl", hash = "sha256:922d7931884fbd5f46785d3e6998c58d1ee3a381b4714d1b5a95da5f41795b7c", size = 2037938 },
+    { url = "https://files.pythonhosted.org/packages/57/33/bd9ee10002c5dacc954624443f2b6ec0d90f1a72ea24479fd70f1c00bba2/pydantic_core-2.34.1-cp311-cp311-macosx_11_0_arm64.whl", hash = "sha256:ca84244021b9d88bd979cb062c58fe7bd00689371b538284f00582781fd1047e", size = 1865764 },
+    { url = "https://files.pythonhosted.org/packages/86/1f/b01bf845616c590895a1edb3daf6749d641209374e65712e2b6174a98fa2/pydantic_core-2.34.1-cp311-cp311-manylinux_2_17_aarch64.manylinux2014_aarch64.whl", hash = "sha256:61f5536849eac65c638e47b7ed69ca9b8f353247936feea3862ff773d0f3e349", size = 1906063 },
+    { url = "https://files.pythonhosted.org/packages/56/a6/fb1f3495dadf291be196e2907b084519e982dd5c762898dcfc05a8bdb80e/pydantic_core-2.34.1-cp311-cp311-manylinux_2_17_armv7l.manylinux2014_armv7l.whl", hash = "sha256:8dbb8babb10c418ca18a149ee27a4df06db5ee4acc30c0576e8c35d9d3a9e9d3", size = 1993222 },
+    { url = "https://files.pythonhosted.org/packages/b1/53/766c15b82e67a8e5f6a26e30c80f9cb18f68145021ecb337e8124b7b7b4c/pydantic_core-2.34.1-cp311-cp311-manylinux_2_17_ppc64le.manylinux2014_ppc64le.whl", hash = "sha256:ec8ce161786818229e96774f4d9159df80589d89b7bc64f3251c57c3fb133289", size = 2142103 },
+    { url = "https://files.pythonhosted.org/packages/04/80/fdb17009b0b78c4cdbd7ce33fc434f45c34f5edd95fed9b3f76b01dc93e1/pydantic_core-2.34.1-cp311-cp311-manylinux_2_17_s390x.manylinux2014_s390x.whl", hash = "sha256:a8d59fcb88752869183b0685e3e6a9d46cb3109280fa452ede7994b9ccff734e", size = 2738738 },
+    { url = "https://files.pythonhosted.org/packages/17/2d/499578ff0d40d411d094fbc35a3d8709dbe7a3e48bc99e6caf9fbbf5ded7/pydantic_core-2.34.1-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl", hash = "sha256:ff39a5611d098fcd821dacdfb01dabb434054867d3df646e7b11981a6770a195", size = 2012075 },
+    { url = "https://files.pythonhosted.org/packages/03/ff/8528ba1261d0d548d88200b9b887633a8599c2baf4326872e7a2d838bdd2/pydantic_core-2.34.1-cp311-cp311-manylinux_2_5_i686.manylinux1_i686.whl", hash = "sha256:6194d491d34ee7667a71cb51f107c9c9266608f86d7fef1e9644acfc7f3e189a", size = 2121560 },
+    { url = "https://files.pythonhosted.org/packages/13/23/7314c1a12db9d3bc7036d94d5c78c81701d9149299f1db0fecbe9a3ed71c/pydantic_core-2.34.1-cp311-cp311-musllinux_1_1_aarch64.whl", hash = "sha256:c59c3cadc8a0ae1ac9bed2d7e78f93e71e5d99bba5f9f121c5d7bd41f1faa89a", size = 2083123 },
+    { url = "https://files.pythonhosted.org/packages/90/75/40f499b653bd745a23e5e7deda21171b559014b41d676ec14f2acc1418fd/pydantic_core-2.34.1-cp311-cp311-musllinux_1_1_armv7l.whl", hash = "sha256:e331d7ec5ec582c5078db948eca25d637c0a7f88a037b5016c48cb7c63f7da2c", size = 2255737 },
+    { url = "https://files.pythonhosted.org/packages/64/b0/6440bbaea7b5bb53dd40e1b725cf06356d01820feb0cb9ffb0b0fd2db125/pydantic_core-2.34.1-cp311-cp311-musllinux_1_1_x86_64.whl", hash = "sha256:71c2d7cb5c9087301f5d2a488ce51e090bd682849b5a59cf4f75a9ce08044f64", size = 2252186 },
+    { url = "https://files.pythonhosted.org/packages/bf/43/6fe3ed2bfbf2068c42d0de892ba5f9b7bdadd494268f56b13bebd592d384/pydantic_core-2.34.1-cp311-cp311-win32.whl", hash = "sha256:dbec17f32828541af5bcebe0ea124235fc620bb5e5d56bf76e38b5b0ad37299d", size = 1917814 },
+    { url = "https://files.pythonhosted.org/packages/c2/57/b39a659a2b5a9c76e239720db257bb1c930349d5094dce93d1b022d2e17b/pydantic_core-2.34.1-cp311-cp311-win_amd64.whl", hash = "sha256:91175524a4cb485333d4119454645c7adefeda34845e66ade277cae670ad8b58", size = 1957753 },
+    { url = "https://files.pythonhosted.org/packages/e5/1d/da4c39b0ee8033f783df8bd7b41e662fadfa8d1be629d74b8e5ff5bfe97e/pydantic_core-2.34.1-cp311-cp311-win_arm64.whl", hash = "sha256:d9cb67f2d7d1f129af7e05dbf4fe209f4df043ea1245086bce2a5ee00ca2f5cc", size = 1905663 },
+    { url = "https://files.pythonhosted.org/packages/ee/44/be71fcdaabc51210171ad47ba58667f5389b55029f2c04a8e009b0cd3bd6/pydantic_core-2.34.1-cp312-cp312-macosx_10_12_x86_64.whl", hash = "sha256:406f85849878788a3fd4b73410955dbbb038a70a8c0d7bb9201084242b035bb5", size = 2019724 },
+    { url = "https://files.pythonhosted.org/packages/3f/dd/c8d5dafae46eacd3a619fe7476794acbbfc5ec170993bce4aed1e6e5773c/pydantic_core-2.34.1-cp312-cp312-macosx_11_0_arm64.whl", hash = "sha256:a953737ad0ce8a6b0130fa77851ab639c576e2c11e16a417392e04f8ec1d40dd", size = 1851663 },
+    { url = "https://files.pythonhosted.org/packages/e6/3a/5f64080cba0002600b7254ecb61a54fbff3ef74023322e3625d95ae8a44a/pydantic_core-2.34.1-cp312-cp312-manylinux_2_17_aarch64.manylinux2014_aarch64.whl", hash = "sha256:e12b71b0ed28cc751e59653d0415aded476f255984dbf0f99d6880342d1911db", size = 1889055 },
+    { url = "https://files.pythonhosted.org/packages/2b/86/6ef6b3820fd52cba959e1b1b109734de99339fcbe78de6f22cb84a1ba568/pydantic_core-2.34.1-cp312-cp312-manylinux_2_17_armv7l.manylinux2014_armv7l.whl", hash = "sha256:c6fe9cb158200d9a83a38fed3c2cb6766a98adc5f61940ccdd41a75231762e89", size = 1976847 },
+    { url = "https://files.pythonhosted.org/packages/7c/93/e4d8c3ce244a528e226e4276c973f1b222ca3bc09e3660506f0b707896aa/pydantic_core-2.34.1-cp312-cp312-manylinux_2_17_ppc64le.manylinux2014_ppc64le.whl", hash = "sha256:448a11df0edc8a1a4072a13d34ad7e048e7c7941635432ffc39036e8bf4454a3", size = 2128118 },
+    { url = "https://files.pythonhosted.org/packages/5e/91/a05cfdaebc9b0e72848dd20b0461a2f3d70cf7578763db8bf077031a83b1/pydantic_core-2.34.1-cp312-cp312-manylinux_2_17_s390x.manylinux2014_s390x.whl", hash = "sha256:642ca6940d2bc027d8ac8ddd4c1c7e7ae3dbb0277e26ab2c42a562cc96f9f048", size = 2686857 },
+    { url = "https://files.pythonhosted.org/packages/37/c7/aa7eb3c22a12fbac158835c5fbd62d2059405838c977a61ba90fb3d9214a/pydantic_core-2.34.1-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl", hash = "sha256:e7126ea9858f9c23a3ec7cd9ecc18be93d1546fe76bc7b331389caf29c4d411f", size = 2013073 },
+    { url = "https://files.pythonhosted.org/packages/0a/14/98d941d9af7d0860884223919c513d2b3b87eec121f3fc9d8864e984ea01/pydantic_core-2.34.1-cp312-cp312-manylinux_2_5_i686.manylinux1_i686.whl", hash = "sha256:d74a8e53545bcfd0515754ece2b018fabfb81c0e49d825f8cc1dba09c911f404", size = 2107513 },
+    { url = "https://files.pythonhosted.org/packages/e9/97/84d36d0c8f92c84ca515e634838f4db29db336d14a65350274ec39927609/pydantic_core-2.34.1-cp312-cp312-musllinux_1_1_aarch64.whl", hash = "sha256:edc0c6d41431849c6f2f6301f713af730884136c06613a41fd6778446b5bc7c5", size = 2066685 },
+    { url = "https://files.pythonhosted.org/packages/2c/5a/1a8d8e3690b595787ab041c0ebab2af6a47dda4df3e087c58ea92cef134c/pydantic_core-2.34.1-cp312-cp312-musllinux_1_1_armv7l.whl", hash = "sha256:f5ee164fb0be1a56c7639794b400815ce36eeb14cdf0a914527c80481b43b43c", size = 2239959 },
+    { url = "https://files.pythonhosted.org/packages/68/72/559845386d05391b5f180711e23efe03811d0b3f9b5a75295da8b2dd8dd4/pydantic_core-2.34.1-cp312-cp312-musllinux_1_1_x86_64.whl", hash = "sha256:b5beed6e1f3ee3d20a93b02bdb28dcc94bd192c09f433cd114efcf8ad13829f0", size = 2241120 },
+    { url = "https://files.pythonhosted.org/packages/57/8b/3ff432a0d84037df43cdca3cb16ba416cb622b619755fe2bbea6b48dd101/pydantic_core-2.34.1-cp312-cp312-win32.whl", hash = "sha256:00d7e7b6a0b2d2294e3dc2d2b170fa860577fc0e286460d7cfd0e90c1a2a1267", size = 1900094 },
+    { url = "https://files.pythonhosted.org/packages/4b/a8/5560b14afe572e6bdc5e7f0067717a7269e45225d9762dd7b57be0babdc0/pydantic_core-2.34.1-cp312-cp312-win_amd64.whl", hash = "sha256:4122151c1f4eb883556adc9e624564731872eb1647f5408028649d69b8565879", size = 1972744 },
+    { url = "https://files.pythonhosted.org/packages/65/ec/64cc90d57bc109f1147f1b3d64439420948a1a0cbee83aeedda6af0d06d0/pydantic_core-2.34.1-cp312-cp312-win_arm64.whl", hash = "sha256:59054ba986777afb760c3283fcf0bfe27a6a1be970f6c7b3bcfde33985a69823", size = 1896488 },
+    { url = "https://files.pythonhosted.org/packages/84/c2/60c310e7d471d590adce553bb0788256ad749bbf4f95f8c1e780e83b4ac6/pydantic_core-2.34.1-cp313-cp313-macosx_10_12_x86_64.whl", hash = "sha256:a7385b01cb068b87b35a95799b4e2b1c6456269d7b9a5e8b4dcfe2547bdaccaf", size = 2026574 },
+    { url = "https://files.pythonhosted.org/packages/e0/51/8f8af7364febb498856bee3f1cc4c4cba45fcaf00622ade3839cb93e8023/pydantic_core-2.34.1-cp313-cp313-macosx_11_0_arm64.whl", hash = "sha256:d26e1ba192c3207fa2a3d1e049ad7710ace1f09e67ae70e52495db70292cc12d", size = 1851446 },
+    { url = "https://files.pythonhosted.org/packages/54/26/93f8fb98f73043e22aa35a3384044694f7fcf2ef7d3031e9ddb47941c4ef/pydantic_core-2.34.1-cp313-cp313-manylinux_2_17_aarch64.manylinux2014_aarch64.whl", hash = "sha256:abdefd076e7857f23caa4cfeacdae831d5e948ac227125b4d35b92b5fb77976c", size = 1893875 },
+    { url = "https://files.pythonhosted.org/packages/35/d3/ae4b5b05b25768252a39f5835a3a4906ca4778a922a2e227a0775ce39c46/pydantic_core-2.34.1-cp313-cp313-manylinux_2_17_armv7l.manylinux2014_armv7l.whl", hash = "sha256:6beb001b3dfc3a6705fd0bec935cf77e99ad37e43f4e82b15495c2c2e678d005", size = 1985569 },
+    { url = "https://files.pythonhosted.org/packages/24/3c/361f8a0edc2b40a746fb19b97d13847e843b474fbcb2654e4df25d1a19a9/pydantic_core-2.34.1-cp313-cp313-manylinux_2_17_ppc64le.manylinux2014_ppc64le.whl", hash = "sha256:502c81a0e3db1010aabdb309456c7217151e94bb20abdbeb766b2df25be84320", size = 2128538 },
+    { url = "https://files.pythonhosted.org/packages/1f/06/edc987266114e25550845c9602f28234721475a7efa84aa9b3cfaafa3d0c/pydantic_core-2.34.1-cp313-cp313-manylinux_2_17_s390x.manylinux2014_s390x.whl", hash = "sha256:df50a223e1a8e8a066a806267b11ffc47fd797736ffafa70734629305b1b6fe6", size = 2687776 },
+    { url = "https://files.pythonhosted.org/packages/c4/a6/a339a95abd8f65ece666147d3457c81abb7c35621c855f02703185b1fa7a/pydantic_core-2.34.1-cp313-cp313-manylinux_2_17_x86_64.manylinux2014_x86_64.whl", hash = "sha256:54e31294764110fc266111f49d5ddd9356773b13fdf848cc261ef07a18de2568", size = 2010400 },
+    { url = "https://files.pythonhosted.org/packages/86/7d/4c4c0e072829588d931162c8371de2cad68f067e9564c8fad49b0516e813/pydantic_core-2.34.1-cp313-cp313-manylinux_2_5_i686.manylinux1_i686.whl", hash = "sha256:e3bb7a0d84fea2629d10adcf46470ef48ac786976f6c3130a441a260beb4dbad", size = 2115352 },
+    { url = "https://files.pythonhosted.org/packages/89/7b/c899ecad0773d1de65e531bb7432886d5d83eeed56e2ccebe0b0434eddeb/pydantic_core-2.34.1-cp313-cp313-musllinux_1_1_aarch64.whl", hash = "sha256:596ea8427dde3721b829ce37e9166efb65c01e25e1977008702b5b081124cec5", size = 2070414 },
+    { url = "https://files.pythonhosted.org/packages/ed/6c/b866f1568d436b5f74289225d9e0a5904c9d56ac4fb1929893e7a05253e4/pydantic_core-2.34.1-cp313-cp313-musllinux_1_1_armv7l.whl", hash = "sha256:b50cd1cd02c525c44d0b4127833436b491f71a6a2ef93d503e29b0b672fe5acb", size = 2248628 },
+    { url = "https://files.pythonhosted.org/packages/09/5b/bd0efb630476c01f46fc9de8af59a1261bd2e17a8d6e70be2166ce68bd42/pydantic_core-2.34.1-cp313-cp313-musllinux_1_1_x86_64.whl", hash = "sha256:a4a44c73e5e58113c1b6a4fe2ab7a151a9d6a0ab612a8e107a6d557c980853f1", size = 2248738 },
+    { url = "https://files.pythonhosted.org/packages/7b/c2/3542e28e1238cef011f0058d156c4473e921f6ea08cd2c4f9bf77bec1492/pydantic_core-2.34.1-cp313-cp313-win32.whl", hash = "sha256:6d908cdc6ee50c463f64792892b7980bad8eea0c231fd09a7db7d8375aeb350e", size = 1906826 },
+    { url = "https://files.pythonhosted.org/packages/2b/ac/713daff8e6df9ac565a44ea066dd37c3a3ae7171ef93329f7c782cff9bae/pydantic_core-2.34.1-cp313-cp313-win_amd64.whl", hash = "sha256:83343e0a9b0e929231bde8d2d7d5dd93dd3af22556bc9a538d729a9ba6824cf4", size = 1971800 },
+    { url = "https://files.pythonhosted.org/packages/99/7d/7df9b739c42e499acc832e6a06fd27b97cc6f374224e9053140c2245edc0/pydantic_core-2.34.1-cp313-cp313-win_arm64.whl", hash = "sha256:92757e71447ff68fa63091c5c4a5f595225fba3f427e987516db56f39568b38e", size = 1903059 },
+    { url = "https://files.pythonhosted.org/packages/c1/ee/9db88f26f7f284550dfa7467533e37392ed007b641d95e34bc259ad00895/pydantic_core-2.34.1-cp313-cp313t-macosx_11_0_arm64.whl", hash = "sha256:32ff9e9e07f80815115466a586f5b437679802cdc79ff58935fc0a4b4b6de77f", size = 1811340 },
+    { url = "https://files.pythonhosted.org/packages/bd/83/8c64954765295648af41130f04fb7f490c11e89cdfe1f06a9767fa8bf485/pydantic_core-2.34.1-cp313-cp313t-manylinux_2_17_x86_64.manylinux2014_x86_64.whl", hash = "sha256:00643ebd798a05111218e9fe17f8b0f782e7cb94cd427b3118938563a81872a5", size = 1986955 },
+    { url = "https://files.pythonhosted.org/packages/ac/41/eaa43c0b1a506aba0fae4dae8095cb363caf5b79e9aad284f09aafd2b0eb/pydantic_core-2.34.1-cp313-cp313t-win_amd64.whl", hash = "sha256:b1f5392e6a615a93f8ba2f71454c096798dd4d944ca27bbca793cfb0e3faaf1d", size = 1942120 },
+    { url = "https://files.pythonhosted.org/packages/2d/4a/0500862bd02a0c0f6517a266dddd587e299a4ac9cf6715b1a10565fa3078/pydantic_core-2.34.1-cp39-cp39-macosx_10_12_x86_64.whl", hash = "sha256:8a7b34f16772a7b02944c1910125fb0b0c89d34c5045e6ea0746044722b592d3", size = 2038228 },
+    { url = "https://files.pythonhosted.org/packages/60/1a/3cbff2c0ad96f78b38118dc6a347c0e12a7aede8dd9ad67d52651b5fc9f4/pydantic_core-2.34.1-cp39-cp39-macosx_11_0_arm64.whl", hash = "sha256:6e1af10b5c6b91732bc8385b472b8b68c120dab52564a0f195409a7020ad3f58", size = 1872870 },
+    { url = "https://files.pythonhosted.org/packages/e1/cd/7a6bb217ad3b7f730dff4abd36f1e7cf6032025408d15dcef01e0d3d9a97/pydantic_core-2.34.1-cp39-cp39-manylinux_2_17_aarch64.manylinux2014_aarch64.whl", hash = "sha256:03c1722c3755b7fbaafcf7e8d331803c6109f96b4df52eb91ea9e1e5f24ee96a", size = 1906430 },
+    { url = "https://files.pythonhosted.org/packages/d6/b2/54ab389304567b841d6f366e6da2ca53e9b0c923da4029da351c9a545dc3/pydantic_core-2.34.1-cp39-cp39-manylinux_2_17_armv7l.manylinux2014_armv7l.whl", hash = "sha256:967a7dc51597a1cc6a0782abf069e549e183101cb3c129a5a37a8e12a956306f", size = 1992407 },
+    { url = "https://files.pythonhosted.org/packages/3c/8f/50e4985198fd00af37c697d4d8b98eaf8a2abf5e28a9e4c8945d964c3e97/pydantic_core-2.34.1-cp39-cp39-manylinux_2_17_ppc64le.manylinux2014_ppc64le.whl", hash = "sha256:b20ff6e14af8c5e3073304c4ec818a9f20bd92ce9d6d5716982803a5bb0851b6", size = 2142490 },
+    { url = "https://files.pythonhosted.org/packages/8d/51/ba611f3d687e58fa85246edaa806a916e1694afaeba7bce540504e2b23ad/pydantic_core-2.34.1-cp39-cp39-manylinux_2_17_s390x.manylinux2014_s390x.whl", hash = "sha256:c21506d9761d51972d3408c45e38ab44e7bdd244cc7e1e2fcd4af0cd6322cb33", size = 2744301 },
+    { url = "https://files.pythonhosted.org/packages/f9/09/39e701ce1a5ae6548ac41922ed42da1511827a835a21ed1d5a461e49fb0a/pydantic_core-2.34.1-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl", hash = "sha256:57e914ce885275b5c9d9641aabc30f2e70945eec65e0e1a915b0242c58a0a5fe", size = 2016408 },
+    { url = "https://files.pythonhosted.org/packages/c1/03/ce681227b9d4ad410936b2a6c0cb20cf201a2d0332c337828440b31338f9/pydantic_core-2.34.1-cp39-cp39-manylinux_2_5_i686.manylinux1_i686.whl", hash = "sha256:390c1c17c3eb47ccb0180af002881294d87203fd01ab617902a6eb740713c017", size = 2121256 },
+    { url = "https://files.pythonhosted.org/packages/cf/55/8a9cc2104acb11b76de2765c7969bfc79bd7d86b89cb07d783ed14d68f3b/pydantic_core-2.34.1-cp39-cp39-musllinux_1_1_aarch64.whl", hash = "sha256:d38504ecbf67b7fe60ae2db028e1b1a54df92a2b899628c82b3edfe885a88059", size = 2083871 },
+    { url = "https://files.pythonhosted.org/packages/4b/89/07b245b60c3add551d20338bd6cda7005be98f5c8ec1062834726fe89f66/pydantic_core-2.34.1-cp39-cp39-musllinux_1_1_armv7l.whl", hash = "sha256:b010fd1d65b536552fc07633b84538300847dc0544ebfea06986017909b4b65e", size = 2254926 },
+    { url = "https://files.pythonhosted.org/packages/77/cc/6cef2ec380e57e20bae11c92495267cb46cff1df28a7a6e278b92532ed0c/pydantic_core-2.34.1-cp39-cp39-musllinux_1_1_x86_64.whl", hash = "sha256:8da35468c5b54142f503b2a7edb0a2252cb6e21206371ab56cb709f064215500", size = 2253401 },
+    { url = "https://files.pythonhosted.org/packages/81/60/10f517468900879f6dc0afae282d278d0b793e623858210e25933ae1d01c/pydantic_core-2.34.1-cp39-cp39-win32.whl", hash = "sha256:4b667f9285c0ffccd17d1ac668da776c5da4c3957391c98a0c4c8ff5f378f040", size = 1918322 },
+    { url = "https://files.pythonhosted.org/packages/5f/58/1e0cfd39b72a52fed5220b5ed5bbba1924d1c0c3cce5e0836a693bb6e26d/pydantic_core-2.34.1-cp39-cp39-win_amd64.whl", hash = "sha256:648a65762994cab52b14c40058290fe0dbbd2ce907b42d7ab7b565fffcfc8617", size = 1962585 },
+    { url = "https://files.pythonhosted.org/packages/bf/f8/50803d0099af8f490bb6bcdecbe220df24c6472f19bc6adbc4af4ca5103a/pydantic_core-2.34.1-pp310-pypy310_pp73-macosx_10_12_x86_64.whl", hash = "sha256:66406c996569bfaa9d6938ccbc43695bc19ee3185dd7ba3e08615e04ca3a0fbe", size = 2035023 },
+    { url = "https://files.pythonhosted.org/packages/33/1a/6073fd39aa6460d6890a3a84e277393b45765f2c872fd633374b3c6d08f7/pydantic_core-2.34.1-pp310-pypy310_pp73-macosx_11_0_arm64.whl", hash = "sha256:a1215a14a145630ce00135d9ea696b5c587f0d166579cef541e8743530dbe134", size = 1867929 },
+    { url = "https://files.pythonhosted.org/packages/b0/4b/4f96ef0786bc7176f26e67f809e7350ea887598a2372b9e69bb9fec00f48/pydantic_core-2.34.1-pp310-pypy310_pp73-manylinux_2_17_aarch64.manylinux2014_aarch64.whl", hash = "sha256:bb5882ad329cec0b58f006840be4b981938303de5ec72e61160b66f86515bd58", size = 1900902 },
+    { url = "https://files.pythonhosted.org/packages/5d/ca/c9e790808f0b217234f79c8a379c5783a6cc5350504e614f24f01b0d66aa/pydantic_core-2.34.1-pp310-pypy310_pp73-manylinux_2_17_x86_64.manylinux2014_x86_64.whl", hash = "sha256:215e185af1ace9682dcaf3bde93d2c78256016ebcfb736b47ed4f58defcfd7bf", size = 2077429 },
+    { url = "https://files.pythonhosted.org/packages/6b/b8/f1cb581892dc96820967a894a0205ebf405f3aff73a08dea6312d223c5a7/pydantic_core-2.34.1-pp310-pypy310_pp73-manylinux_2_5_i686.manylinux1_i686.whl", hash = "sha256:911024e92269e0da607eeee32ab5f6e460e8afe74e0859a353d8f77a09440ab2", size = 2116895 },
+    { url = "https://files.pythonhosted.org/packages/e4/37/555f5cbe5d62457260e85cf91525bd1186f697303db0056d9ae664292cdc/pydantic_core-2.34.1-pp310-pypy310_pp73-musllinux_1_1_aarch64.whl", hash = "sha256:b1073b3688aa0b1834e54a2e4f1aaa72306bbe72138983a0bd4bf3c8ac9751d5", size = 2077587 },
+    { url = "https://files.pythonhosted.org/packages/b5/a0/b0a359ee9b3bb9d0a838b9891bd3f5d2ff3afe993500860c4b29111c6812/pydantic_core-2.34.1-pp310-pypy310_pp73-musllinux_1_1_armv7l.whl", hash = "sha256:16c73d64c4ff64a8c107bd779dfc957f6d5add4f14d348441a836ec23d620cf4", size = 2247416 },
+    { url = "https://files.pythonhosted.org/packages/33/cf/b03a86dbc7f00fdac44ac11045ceb80db04fa91a477fa23562ea1a1bc142/pydantic_core-2.34.1-pp310-pypy310_pp73-musllinux_1_1_x86_64.whl", hash = "sha256:ca7d1b95b1bca481fb6a8c0ae684a053eef0d6da9e819f6361fd1da7bc1c54ad", size = 2248907 },
+    { url = "https://files.pythonhosted.org/packages/a4/88/7031a9395b20093e128676bf697b6dc96c07b21ae2c5611ef4e70fad8631/pydantic_core-2.34.1-pp310-pypy310_pp73-win_amd64.whl", hash = "sha256:286981cc3a187e128d5223f37b5660dcf7021f16f4164dee28c4ff78e0fcd040", size = 2075820 },
+    { url = "https://files.pythonhosted.org/packages/04/b2/27724854768e29f8f6c9975dc94851cbf579ef4ca6063e046b7a91ffb79f/pydantic_core-2.34.1-pp311-pypy311_pp73-macosx_10_12_x86_64.whl", hash = "sha256:7c848c98ceaf3d1d6369814565da0b2f4aa1cd8b3bf28767261f54a13b8b387d", size = 2036183 },
+    { url = "https://files.pythonhosted.org/packages/c6/18/ff9d9e91d333ba93ea55477196b28f9cbc712668a84817c1ae5da10bb69f/pydantic_core-2.34.1-pp311-pypy311_pp73-macosx_11_0_arm64.whl", hash = "sha256:c80e6d594122345b41659e5d5ad9cb5d732d81a1da83c336e1b13f55794abafe", size = 1868773 },
+    { url = "https://files.pythonhosted.org/packages/10/49/5ec5bb4c7c530853e54ec4f1fde0ec216c44abb3b7b70c8e2e63e21b60c3/pydantic_core-2.34.1-pp311-pypy311_pp73-manylinux_2_17_aarch64.manylinux2014_aarch64.whl", hash = "sha256:64b8fa876a63cf806ad0d13dc7205b47512c90f192df2c1ad6a7f49a90015003", size = 1900859 },
+    { url = "https://files.pythonhosted.org/packages/93/84/9ee32d2468d0f7df469d7f927d63950ebbf4493898c4f8e88f363c505828/pydantic_core-2.34.1-pp311-pypy311_pp73-manylinux_2_17_x86_64.manylinux2014_x86_64.whl", hash = "sha256:9c2d8176efe66d54a5f7d3052b394ede8c0d2bb80144d1bf6c9b30a9d1fdfaea", size = 2077388 },
+    { url = "https://files.pythonhosted.org/packages/01/56/b43444372fb9fd88ae867d907b184b8776e353dacf6d4c1f80a350bc4f82/pydantic_core-2.34.1-pp311-pypy311_pp73-manylinux_2_5_i686.manylinux1_i686.whl", hash = "sha256:69eb11a0bcbdb49c04494980206e9449d361b78b35ad64478d9ab15db837bec9", size = 2117737 },
+    { url = "https://files.pythonhosted.org/packages/a0/37/1b5da00af59ed09eafebda1d8bc3c816b2777b64d054f9203d05c6445630/pydantic_core-2.34.1-pp311-pypy311_pp73-musllinux_1_1_aarch64.whl", hash = "sha256:b30d9d48390deb789c84205fc76ad6c27b00819d60dc955186e9387c55370413", size = 2077383 },
+    { url = "https://files.pythonhosted.org/packages/2e/ac/de6883ac0617243370cfed67ba785aaae27bea5e3369ed329ddadbba360b/pydantic_core-2.34.1-pp311-pypy311_pp73-musllinux_1_1_armv7l.whl", hash = "sha256:e3da84e67845c83e88c7e96df3e487a16752753f19241d17e887f1f39b7a851c", size = 2248548 },
+    { url = "https://files.pythonhosted.org/packages/c3/9e/3b4f95aca1742eaa064e402f2388a2f1013f2214f7fb0f4f6155eff3dec3/pydantic_core-2.34.1-pp311-pypy311_pp73-musllinux_1_1_x86_64.whl", hash = "sha256:9b3946e2fb4a7821b59a70193530f463dd532fd6d25f13e9c6521df736906777", size = 2248760 },
+    { url = "https://files.pythonhosted.org/packages/f8/4d/06e5db4b171a708c35149b475aa8823a3553c9d4b8b3d6afc1736d5665bc/pydantic_core-2.34.1-pp311-pypy311_pp73-win_amd64.whl", hash = "sha256:46b2ae1824bb07939f77d4a390a1a0a5b0838e34ac9427cf21c89a7a925f1ad7", size = 2075955 },
+    { url = "https://files.pythonhosted.org/packages/01/a7/3d3347ef8685efd48f0f9044e2ab44a33865ba2bd48e9c297aab9990a636/pydantic_core-2.34.1-pp39-pypy39_pp73-macosx_10_12_x86_64.whl", hash = "sha256:9fd3ae017ad680b744ffe257627c3503889d2a6ef2b7d1493483e9faccf7abc7", size = 2035248 },
+    { url = "https://files.pythonhosted.org/packages/7c/00/3f8e0632c27ce1d969b5e9caf8d5d609719c2951df786ad5616f7e8c1b34/pydantic_core-2.34.1-pp39-pypy39_pp73-macosx_11_0_arm64.whl", hash = "sha256:18ff148029c246e3ca36164ca07fa3d0d7c99d2feab98e12bce97da63e67bcdb", size = 1868441 },
+    { url = "https://files.pythonhosted.org/packages/0d/0a/2f02f1e1aed3d3aebbabe560a6e0ca5de7ea080aeebded29a4c5cf05c99d/pydantic_core-2.34.1-pp39-pypy39_pp73-manylinux_2_17_aarch64.manylinux2014_aarch64.whl", hash = "sha256:f7c7ac737d1544615403e8a98956fc02b7505f713c531eab7c4b265618e82af3", size = 1901157 },
+    { url = "https://files.pythonhosted.org/packages/0e/cd/f601db334adf27808b6179415538cc3581f542daacd3eb931a3b32d1a100/pydantic_core-2.34.1-pp39-pypy39_pp73-manylinux_2_17_x86_64.manylinux2014_x86_64.whl", hash = "sha256:0d8f64e49175f98416b7193c818741916dc29783256f684324cee262a1b35d57", size = 2078186 },
+    { url = "https://files.pythonhosted.org/packages/b2/35/7c5bf738ce45ad9943aca8d6fde04db9e43c5cb17b7248e2d07cb1546458/pydantic_core-2.34.1-pp39-pypy39_pp73-manylinux_2_5_i686.manylinux1_i686.whl", hash = "sha256:e19bb6d38d5259b20de62618928fe33d60c4b15975f891503e0703776d23e9f7", size = 2117064 },
+    { url = "https://files.pythonhosted.org/packages/bb/5d/ff9d24da5991f22ce44e6b101c672a439f97be1f3f0f433334d945ae7aa2/pydantic_core-2.34.1-pp39-pypy39_pp73-musllinux_1_1_aarch64.whl", hash = "sha256:ee4491ebcee7f58054585fedeaca653e9822b97445e9b1700ed29846a963f210", size = 2078212 },
+    { url = "https://files.pythonhosted.org/packages/0b/b7/71cda0671a13529f9d5d2b2c92113e40eb759cdcc661abd921393344ed47/pydantic_core-2.34.1-pp39-pypy39_pp73-musllinux_1_1_armv7l.whl", hash = "sha256:08a5b5da50a28a5bae06120aaae7ff086b19a870c74f6ab110326893252822f8", size = 2247591 },
+    { url = "https://files.pythonhosted.org/packages/c0/20/2a2199fd7b2b3b238df64bf42198450a45d7891a1115e539a99d6b4ed5d8/pydantic_core-2.34.1-pp39-pypy39_pp73-musllinux_1_1_x86_64.whl", hash = "sha256:a85f3126af1cf983f252738209713f625982ede1557e1991feb6e817c1116b23", size = 2249304 },
+    { url = "https://files.pythonhosted.org/packages/ea/09/3404f068c7b4884e6c9c557dac52a04e575b1bf25911e99a70b34ac2daf7/pydantic_core-2.34.1-pp39-pypy39_pp73-win_amd64.whl", hash = "sha256:6074b7d2bc969caf9669303e815f52a448ed8f107f17e2905474e7b956a64481", size = 2076010 },
 ]
 
 [[package]]
